artifactory/commands/transferfiles/manager.go

```diff
@@ -182,19 +190,46 @@ func (ptm *PollingTasksManager) stop() {
 }
 
 type producerConsumerWrapper struct {
-	chunkBuilderProducerConsumer  parallel.Runner
+	// This Producer-Consumer is used to upload chunks, initialized in newProducerConsumerWrapper; each uploading thread waits to be given tasks from the queue.
 	chunkUploaderProducerConsumer parallel.Runner
-	errorsQueue                   *clientUtils.ErrorsQueue
+	// This Producer-Consumer is used to build chunks and execute AQL tasks. The chunks data is sent to the threads that will upload them.
```

Comment:
...is used to build chunks and execute AQL tasks...
-->
...is used to execute AQLs and build chunks from the AQLs' results...

...to the threads that...
-->
...to the go routines that...

Reply:
done